Makefile

```diff
@@ -135,6 +135,9 @@ test-e2e: build-e2e-all
 test-install-e2e: build-e2e-all
 	hack/run-test-install-e2e.sh
```

Comment:
do we need changes to this wrapper script, perhaps just changing the defaults?
it seems to me this PR is not separating, rather adding a new targeted suite.

Reply:
My bad! forgot to delete that part from the install script.

Reply:
Updated the code to reflect this.
Sorry for the confusion.